app/src/main/AndroidManifest.xml

```diff
@@ -4,6 +4,10 @@
     package="ch.epfl.qedit">
 
     <uses-permission android:name="android.permission.CAMERA"/>
+    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION"/>
+    <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION"/>
```

Comment:
What are `ACCESS_FINE_LOCATION`, `ACCESS_COARSE_LOCATION`, `ACCESS_NETWORK_STATE`. What are they used for ? Please add some comments.